src/main/resources/io/seqera/wave/spack/spack-builder-singularityfile.txt

```diff
@@ -32,7 +32,21 @@ EOF
     spack config add concretizer:unify:true
     spack config add concretizer:reuse:true
     spack config add packages:all:target:[{{spack_arch}}]
+    spack -e . config add packages:all:providers:blas:[openblas]
```

Comment:
why these uses `spack -e . config add` and the other `spack config add` ?

Reply:
they all need to be with `-e .`, but changes for the other lines are in the sister PR , https://github.com/seqeralabs/wave/pull/368

Reply:
is it better if I merge both PRs into one? this will also make integration testing simpler - test only once